### ISSUE TITLE
Voice over is not announcing the name of created topic in grid view.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-iOS-SDK#707]
Category: Screen Reader
MAS Rule: MAS4.2.1
Repro Steps
1. Launch Embedded Social Test App.
2. Navigate to Hamburger button and double tap.
3. Navigate to hamburger button and double tap.
4. Navigate to the Profile link and double tap.
5. Navigate to the Icon gallery button and double tap.
6. Navigate the page via swipe gesture.
7. Verify that speech of voice over for the created topics.
Expected:
Voice over should announce the name of created topics in grid view. It should not announce the image-image for all created topics, It should announce topic name and role also. So that screen reader convey proper information regarding the topic.
Actual:
Voice over is not announcing the name of created topic in grid view. It is just announcing the image image for all created topics.
Test Environment:
OS: iOS 11.0.2
App: Embedded Social Test
App Version: 0.7.8 (8)
Tools: Voice Over, Keyboard
